.template-lintrc.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
   plugins: ["ember-template-lint-plugin-discourse"],
   extends: "discourse:recommended",
+
+  rules: {
+    "no-capital-arguments": false, // TODO: we extensively use `args` argument name
```

Comment:
what case do we have with capital arg?

Reply:
The plugin outlet component uses the `@args` attribute, and this rule complains about it. It's already disabled in core, so I'm doing the same here.

Reply:
hmmm, it's not supposed to be the case AFAIK, this rule is preventing this: `<Foo @Bar={1}>`

https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-capital-arguments.md

Reply:
It also complains about the `@args` attribute being restricted. See [here](https://github.com/discourse/discourse-chat/actions/runs/3323743368/jobs/5494479287#step:8:11)